post: v1.2.0 (#52)
rero/rero21.ch: content/post/sortie-de-la-v1.2.0.en.md

---
title: "V1.2.0 is out: thanks for the translations!"
date: 2021-05-19
draft: false
tags: ["multilinguism", "release", "rero-ils"]
---

> After another intense sprint at RERO+ and UCL Louvain, the arrival of version `v1.2.0` brings its share of new features and fixes to stabilize RERO ILS before the *go-live* on July 12th. The complete release notes are available on [GitHub](https://github.com/rero/rero-ils/blob/dev/RELEASE-NOTES.rst#v120). This is an opportunity for us to highlight the community aspect of RERO ILS with the system's translation processes.

Reviewer: ```suggestion
After another intense sprint at RERO+ and UCLouvain, the arrival of version `v1.2.0` brings its share of new features and fixes to stabilize RERO ILS before the *go-live* on July 12th. The complete release notes are available on [GitHub](https://github.com/rero/rero-ils/blob/dev/RELEASE-NOTES.rst#v120). This is an opportunity for us to highlight the community aspect of RERO ILS with the system's translation processes.
```

<!--more-->

## RERO ILS community translation

In order to be open and accessible, RERO ILS must also be multilingual. In this perspective, the software is translated with the help of the [Weblate service](https://hosted.weblate.org/projects/rero_plus/rero-ils/), which allows the development team as well as external people to collaborate. Indeed, the community can submit translations for the project. We would like to thank the following members for their contributions: 

- [Ambros Gattlen](https://hosted.weblate.org/user/ambgat/)
- [phlostically](https://hosted.weblate.org/user/phlostically/)
- [Adolfo Jayme Barrientos](https://hosted.weblate.org/user/Fito/)
- [Allan Nordhøy](https://hosted.weblate.org/user/kingu/)

> If you also want to contribute, please visit [the project page on Weblate](https://hosted.weblate.org/projects/rero_plus/#information).

Reviewer: ```suggestion
If you would like to contribute as well, please visit [the project page on Weblate](https://hosted.weblate.org/projects/rero_plus/#information).
```

## Several organizations for one patron

> Among the many new features of `v1.2.0`, the ability for a patron to be registered in the libraries of multiple organizations was a clear need of the RERO+ libraries' infrastructure. 

Reviewer: ```suggestion
Among the many [new features](https://github.com/rero/rero-ils/blob/dev/RELEASE-NOTES.rst#v120) of `v1.2.0` is the possibility of registering at libraries of multiple organizations using the same account, which is clearly a requirement of patrons of the RERO+ libraries. 
```

<!--more-->

## RERO ILS community translation

In order to be open and accessible, RERO ILS must also be multilingual. In this
perspective, the software is translated with the help of the [Weblate
service](https://hosted.weblate.org/projects/rero_plus/rero-ils/), which allows
the development team as well as external people to collaborate. Indeed, the
community can submit translations for the project. We would like to thank the
following members for their contributions:

- [Ambros Gattlen](https://hosted.weblate.org/user/ambgat/)
- [phlostically](https://hosted.weblate.org/user/phlostically/)
- [Adolfo Jayme Barrientos](https://hosted.weblate.org/user/Fito/)
- [Allan Nordhøy](https://hosted.weblate.org/user/kingu/)

If you would like to contribute as well, please visit [the project page on
Weblate](https://hosted.weblate.org/projects/rero_plus/#information).

## Several organizations for one patron

Among the many [new features][release-notes] of `v1.2.0` is the possibility of
registering at libraries of multiple organizations using the same account,
which is clearly a requirement of patrons of the RERO+ libraries.

> As previously [stated](/nouvelle-version-1.1.0/), user traffic data is not

Reviewer 2: ```suggestion
As previously [stated](/nouvelle-version-1.1.0/), user circulation data is not
```

shared between different organizations; however, it is likely that individuals

> are registered in libraries of different organizations. Patrons can now view

Reviewer 2: On aurait pu rappeler ici par un exemple concret ce que sont différentes organisations... Les gens ne connaissent pas forcément ce terme de jargon RERO ILS.

> it is likely that individuals are registered in libraries of different organizations, for instance once in the RERO Valais network and once in the RBNJ (Neuchâtel-Jura) network.

On aurait aussi pu mettre une note de bas de page l'expliquant:

> An organization in RERO ILS is a group of libraries working closely together, especially by sharing common circulation rules. In RERO+ for instance, all Valais libraries are grouped in one organization, and all Neuchâtel-Jura libraries in another.

their data (loans, history, fines, etc.) from their different libraries through

> a dropdown menu on their account page. The patron does not need to switch

Reviewer 2: ```suggestion
a dropdown menu on their account page. They don't need to switch
```
